redirector.k8s.io/README.md

```diff
@@ -0,0 +1,36 @@
+Overview
+====
+This contains the TXTDirect configuration for redirector.k8s.io backend service.
+
+Currently used by:
+- go.k8s.io
+- go.kubernetes.io
+
+Vanity URL(s)
+====
+
+|  | k8s.io | kubernetes.io |
+| --- | --- | --- |
+| Redirections | https://go.k8s.io | https://go.kubernetes.io |
+
+Redirections
+====
+- https://go.k8s.io/api-review
```

Comment:
I don't think we should document them here - just let the DNS config be canonical?